SEED-platform/seed
# inventory reports page uses local storage to show previous selections

#### Any background context you want to provide?
See tagged issue

#### What's this PR do?
Add report settings to cache and use it on page load. Also, render the charts whenever this page is loaded.

#### How should this be manually tested?
Use the inventory reports page. Leave, remembering the settings you last used, then revisit the page. See that the page has your previous selections and also reloads the page.

#### What are the relevant tickets?
#417 

#### Screenshots (if appropriate)
